src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -171,8 +171,8 @@ public class LivingEntityGUI extends ModElementGUI<LivingEntity> {
 	private final JTextField mobLabel = new JTextField();
 
 	private final JCheckBox spawnInDungeons = L10N.checkbox("elementgui.living_entity.spawn_dungeons");
-	private final JColor spawnEggBaseColor = new JColor(mcreator);
-	private final JColor spawnEggDotColor = new JColor(mcreator);
+	private final JColor spawnEggBaseColor = new JColor(mcreator, false, true);

```

Comment:
Here, transparency is not allowed, same for potion offect and many other cases

Reply:
Then what did you mean by most cases that
>need to be limited in transparency

?

Reply:
>I would... only enable transparency when really needed

So it should only be possible for image editor things?

Reply:
Very likely, yes

Reply:
:heavy_check_mark: Fixed